### PR TITLE
Created html and js files which helps fetch messages and present them in a user-friendly way

### PR DESCRIPTION
**Pull request by archthegit**

### CODE REVIEW

File: src/main/webapp/js/feed.js [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+  const messageDiv = document.createElement('div');
+  messageDiv.classList.add('message-div');
+  messageDiv.appendChild(headerDiv);
+  messageDiv.appendChild(bodyDiv);
+
+  return messageDiv;
+}
+
+// Fetch messages and add them to the page.
+function fetchMessages() {
```
Comment (SuyashLakhotia) [inline src/main/webapp/js/feed.js]: Add "// eslint-disable-line no-unused-vars" here.

See example: https://github.com/codeu-summer-2019-team-51/project/blob/master/src/main/webapp/js/navigation-loader.js#L45

File: src/main/webapp/js/feed.js [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+  const url = '/feed';
+  fetch(url).then((response) => { return response.json(); })
```
Comment (SuyashLakhotia) [inline src/main/webapp/js/feed.js]: Change to `fetch(url).then(response => response.json()).then(...)`

File: src/main/webapp/js/feed.js [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+  .then((messages) => {
```
Comment (SuyashLakhotia) [inline src/main/webapp/js/feed.js]: For indentation, I use Atom + atom-beautify (after setting the indentation settings). :)

Comment (SuyashLakhotia) [reply src/main/webapp/js/feed.js]: Also takes care of spacing. :)

File: src/main/webapp/feed.html [hunk 4]
```diff
@@ -0,0 +1,20 @@
...
+<html>
+
+<head>
+  <title>Message Feed</title>
+  <link rel="stylesheet" href="/css/main.css">
+  <link rel="stylesheet" href="/css/user-page.css">
+  <script src="js/feed.js"></script>
+</head>
+
+<body onload="buildUI()">
```
Comment (SuyashLakhotia) [inline src/main/webapp/feed.html]: Missing semicolon: "buildUI();"

File: src/main/webapp/feed.html [hunk 5]
```diff
@@ -0,0 +1,20 @@
...
+  <div id="content">
+    <h1>Message Feed</h1>
+    <hr />
+    <div id="message-container">Loading...</div>
+  </div>
+
+</body>
+
+</html>
```
Comment (SuyashLakhotia) [inline src/main/webapp/feed.html]: Missing newline at the end of file (this is useful for future diffs to be clean).

File: src/main/webapp/js/feed.js [hunk 6]
```diff
@@ -0,0 +1,48 @@
...
+  const messageDiv = document.createElement('div');
+  messageDiv.classList.add('message-div');
+  messageDiv.appendChild(headerDiv);
+  messageDiv.appendChild(bodyDiv);
+
+  return messageDiv;
+}
+
+// Fetch messages and add them to the page.
+function fetchMessages() { // eslint-disable-line no-unused-vars
```
Comment (SuyashLakhotia) [inline src/main/webapp/js/feed.js]: This probably doesn't need the `no-unused-vars` comment anymore?

File: src/main/webapp/js/feed.js [hunk 7]
```diff
@@ -0,0 +1,48 @@
+function buildMessageDiv(message) {
+  const usernameDiv = document.createElement('div');
+  usernameDiv.classList.add('left-align');
```
Comment (SuyashLakhotia) [inline src/main/webapp/js/feed.js]: Just checking, are the `.left-align` & `.right-align` CSS classes defined? Working as expected?

Comment (archthegit) [reply src/main/webapp/js/feed.js]: Just noticed these two aren't defined! I didn't look into it cause the messages appeared the same way it was shown on the codeU website. I will add these in, thanks for pointing it out!

File: src/main/webapp/css/user-page.css [hunk 8]
```diff
@@ -13,7 +13,6 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-
```
Comment (SuyashLakhotia) [inline src/main/webapp/css/user-page.css]: Keep the newline?

File: src/main/webapp/css/user-page.css [hunk 9]
```diff
@@ -40,6 +39,15 @@
   margin: 5px;
 }
 
+.left-align {
+  margin: 5px;
+  text-align: left;
+}
+
+.right-align {
+  margin: 5px;
```
Comment (SuyashLakhotia) [inline src/main/webapp/css/user-page.css]: Does this need `text-align: right`?

**Comment by SuyashLakhotia** [comment]
Approving early but please resolve the two comments before merging!

File: src/main/webapp/css/user-page.css (hunk 8 above)
Comment (archthegit) [reply src/main/webapp/css/user-page.css]: I just thought it looked weird cause there's another comment describing the function after it. But i'll take it out!

File: src/main/webapp/css/user-page.css (hunk 9 above)
Comment (archthegit) [reply src/main/webapp/css/user-page.css]: It does need a text-align. It just makes the whole message box look really weird. How about I keep the function and left-align the date as well?

**Pull request merged**